components/container/index.jsx

```diff
@@ -49,6 +49,32 @@ import { isHTTPError } from "../../src/error";
 import { locationIsConnectedToProfile } from "../../src/solidClientHelpers/profile";
 import { isContainerIri } from "../../src/solidClientHelpers/utils";
 
+function isNotAContainerResource(iri, container) {
```

Comment:
These changes are unrelated to the subject of the PR. Could we please have them in a separate PR?